packages/core/src/stepper/stepper-content.tsx

```diff
@@ -0,0 +1,29 @@
+import { type ValidComponent, splitProps, JSX } from "solid-js";
+import { Tabs } from "../tabs";
+import { type ElementOf, type PolymorphicProps } from "../polymorphic";
+
+export interface StepperContentOptions {
+  index: number;
```

Comment:
Can this index be calculated dynamically? For instance, if I don't need the step triggers at the top and want to have dynamic steps?

To accomplish this, I would use a "register and unregister" pattern to adjust the index dynamically and set it for the tab content.

See for example
- https://github.com/corvudev/corvu/blob/e87c758a5bc64cb81d7a7900bd3ae99abaeb298f/packages/accordion/src/context.ts#L65-L66
- https://github.com/corvudev/corvu/blob/e87c758a5bc64cb81d7a7900bd3ae99abaeb298f/packages/accordion/src/Root.tsx#L243-L246

Reply:
usually, stepper forms are not dynamic. It usually have fixed number of steps. so I thought it would better not consider this edge usecase.

Reply:
Your approach is quite reasonable. In JSX, you can place step components anywhere and manually set their indexes.

However, let's consider a scenario where you have a room booking form. A room may or may not have zones (i.e. it's large or not), and these zones can also be booked.

In this case, if you have a room selection step and want to dynamically add the next step based on whether the selected room has zones, would it be more effective to implement a regular step skip (i.e. `context.setCurrentIndex(context.currentIndex + 2)`) or to dynamically add a new step?

Additionally, how would this look if we use `Stepper.Trigger`s for the steps? How will it be displayed visually? Let's say empty or not completed? In dynamic implementation there are also disadvantages for this: UI can "jump"